scripts/create-hosted-graphite-alerts.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Description:
+    Sets up alerts for missing logs for our applications using the Hosted Graphite alerting API
+
+    https://www.hostedgraphite.com/docs/alerting/alerting_api.html
+
+    Note, this script has been used as a one off to create alerts. If a user wishes to edit existing alerts than this
+    will likely not work as there will be a 409 (conflict) response returned. As this is likely a rare process, it is
+    advised to manually delete alerts that you wish to replace before rerunning this script.
+
+Usage:
+    scripts/create-hosted-graphite-alerts.py <hosted_graphite_api_key>
+
+Example:
+    scripts/create-hosted-graphite-alerts.py apikey
+"""
+import json
+
+import requests
+from docopt import docopt
+
+
+# No staging alert as we have a limit on how many alerts we can have and this will cover both the first step of our
+# pipeline and also production
+ENVIRONMENTS = ["preview", "production"]
+
+APPS = ["api", "search-api", "admin-frontend", "buyer-frontend", "briefs-frontend", "brief-responses-frontend",
+        "router", "supplier-frontend", "user-frontend"]
```

Comment:
Do we not have an existing list of "all apps" we can piggyback on here? I can imagine someone adding a new app (if I grit my teeth :grimacing:) and neglecting to add it here...

Reply:
Ahh, fair point!

Reply:
Good question, let me look into this